src/main/java/duke/Task.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+public class Task {
```

Comment:
Perhaps a high-level javadocs overview of the main functionalities of each public class could be added so users know what the class is primarily used for without having to delve into its methods.